extendr-api/tests/call_tests.rs

```diff
@@ -0,0 +1,27 @@
+use extendr_api::prelude::*;
+
+#[test]
+fn formula_test() {
+    test! {
+        // As one R! macro call
+        let confint1 = R!(confint(lm(weight ~ group - 1, PlantGrowth)))?;
```

Comment:
I think it would be best to put the argument to `R!()` into a string. Let's get into the habit of always doing that.

Reply:
I've changed all the R! macros without quotes I could find to have quotes.